apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -240,6 +241,14 @@ const Dashboard = () => {
         This app allows you to build a custom multisend transaction.
         <br />
         Enter a Ethereum contract address or ABI to get started.
+        <br />
+        <Link
+          href="https://help.gnosis-safe.io/en/articles/4680071-create-a-batched-transaction-with-the-transaction-builder-safe-app"
+          size="lg"
+          target="_blank"
```

Comment:
given the `target="_blank"` shouldn't we add `rel="noreferrer"` too?

Reply:
@nicosampler :point_up_2:

Reply:
pushed! thx